packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte

```diff
@@ -847,7 +835,7 @@
     {#if !isPayee && !unsupportedNetwork && !isPaid && !isRequestPayed && !isSigningTransaction}
       {#if !hasEnoughBalance}
         <div class="balance-warning">
-          Insufficient funds: {userBalance}
+          Insufficient funds: {Number(userBalance).toFixed(4)}
```

Comment:
The problem with Number is that it can fail if amount exceeds MAX_SAFE_INTEGER. But I'll approve anyway.